src/main/java/mate/jdbc/services/impl/DriverServiceImpl.java

```diff
@@ -0,0 +1,45 @@
+package mate.jdbc.services.impl;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.impl.DriverDaoImpl;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+import mate.jdbc.services.DriverService;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private static final DriverDao driverDao = new DriverDaoImpl();
```

Comment:
Remove the assignment of a new DriverDaoImpl() instance to driverDao. The @Inject annotation is used for dependency injection, and it will take care of initializing the driverDao field.